.copywrite.hcl

```diff
@@ -7,6 +7,9 @@ project {
   header_ignore = [
     "command/asset/*.hcl",
     "command/agent/bindata_assetfs.go",
+    "api/**",
+    "drivers/shared/**",
+    "plugins/**",
```

Comment:
Is the copyright bot unable to distinguish between licenses it's supposed to use? We'll definitely be adding files to these in the future, which will need the MPL2 license.